### PR TITLE
Improves andino_navigation package.

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

Related to #190 

## Summary
Goes on top of #223 which facilitates nav2 integration on real robot

 - Add readme to explain `andino_navigation` package use
 - Minor tuning of parameters, mainly reducing inflation radius as the robot is small per sei.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,20 +166,19 @@ ros2 launch andino_bringup rviz.launch.py
...
 The [`andino_navigation`](./andino_navigation/README.md) package provides a navigation stack based on the great [Nav2](https://github.com/ros-planning/navigation2) package.
 
 https://github.com/Ekumen-OS/andino/assets/53065142/29951e74-e604-4a6e-80fc-421c0c6d8fee
 
-_Important!: At the moment this package is only working with the simulation. The support for the real robot is forthcoming._
+Follow the [`andino_navigation`'s README](./andino_navigation/README.md) instructions for bringing up the Navigation stack in the real robot or in the simulation.
 
+## :computer: Simulation
+
+The [`andino_gz_classic`](./andino_gz_classic/README.MD) package provides a Gazebo simulation fo the Andino robot.
```
Comment (jballoffet) [inline README.md:179]: typo: for

**Review by jballoffet — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (francocipollone) [reply README.md]: Done!

**Review by jballoffet — DISMISSED** [review]
LGTM!

**Pull request merged**